includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -134,6 +137,14 @@ public function __construct(
 				'description' => __( 'Charge must be captured within 7 days of authorization, otherwise the authorization and order will be canceled.', 'woocommerce-payments' ),
 				'default'     => 'no',
 			],
+			'saved_cards'                  => [
+				'title'       => __( 'Saved Cards', 'woocommerce-payments' ),
+				'label'       => __( 'Enable Payment via Saved Cards', 'woocommerce-payments' ),
+				'type'        => 'checkbox',
+				'description' => __( 'If enabled, users will be able to pay with a saved card during checkout. Card details are saved on Stripe servers, not on your store.', 'woocommerce-payments' ),
```

Comment:
I'm not sure about mentioning Stripe here. It's clear that we're using Stripe for WCPay, but it might be better to mention something like: `Card details are saved on our platform, not on your store.`. What do you think?

Reply:
I agree, I think it makes it more clear and simple for the user.